torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1176,10 +1185,12 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
             runGraphFunction(stack, &f, &af);
           } break;
           case OP:
+            frames.back().pc = af.pc;
```

Comment:
The point of the active frame is to avoid having to edit the stack on each instruction. Either this change should not be made, or the entire ActiveFrame stuff should simply be removed because it no longer serves a purpose.